archetypes/Pools/FilterSelects/Bar.tsx

```diff
@@ -41,143 +43,175 @@ const REBALANCE_OPTIONS_MOBILE = [
-                    <Filters
-                        className="lg:hidden m-auto w-8 ml-2"
-                        onClick={() => dispatch({ type: 'setFiltersOpen', open: !state.filtersOpen })}
+                    <div className="flex mb-5">
+                        <Dropdown
+                            value={state.leverageFilter}
+                            className="w-32 mt-auto mr-5"
+                            options={Object.keys(LeverageEnum).map((key) => ({
+                                key: (LeverageEnum as any)[key],
+                                ticker: (key !== 'All' ? key : '') as LogoTicker,
+                            }))}
+                            onSelect={(val) => dispatch({ type: 'setLeverageFilter', leverage: val as LeverageEnum })}
+                        />
+                        <SearchInput
+                            className="w-60"
+                            placeholder="Search"
+                            value={state.search}
+                            onChange={(search) => dispatch({ type: 'setSearch', search })}
+                        />
+                    </div>
+                    <TWButtonGroup
+                        value={state.deltaDenotation}
+                        onClick={(option) => dispatch({ type: 'setDenotation', denotation: option as DeltaEnum })}
+                        color="greyed"
+                        border="rounded"
+                        borderColor="greyed"
+                        options={DENOTATION_OPTIONS}
                     />
-                </div>
-                <HiddenExpand className="lg:hidden" defaultHeight={0} open={state.filtersOpen}>
+                </HiddenExpand>
+
+                <div className="mt-5 flex flex-col md:flex-row md:justify-between md:items-center">
                     <div className="flex">
-                        <div className="flex mr-4 flex-col">
-                            <span className="inline-flex">
-                                <ArrowDown className="rotate-180 text-green-600 h-6" />
-                                <ArrowDown className="text-red-600 mr-1 h-6" />
-                                <h3 className="mb-1 text-theme-text">Denotion</h3>
-                            </span>
-                            <div className="mt-auto ml-auto">
-                                <TWButtonGroup
-                                    value={state.deltaDenotion}
-                                    onClick={(option) =>
-                                        dispatch({ type: 'setDenotion', denotion: option as DeltaEnum })
-                                    }
-                                    color="greyed"
-                                    border="rounded"
-                                    borderColor="greyed"
-                                    options={DENOTION_OPTIONS}
-                                />
-                            </div>
+                        <div className="xl:hidden">
```

Comment:
can you align these with the buttons to the right of them 
![image](https://user-images.githubusercontent.com/29347276/153515703-24036852-d9db-4634-8a63-a072e8b23e88.png)
change to 
![image](https://user-images.githubusercontent.com/29347276/153515797-4b3262bd-44b5-4a07-9b15-35e15ca670ce.png)

Reply:
✅